database/buildSchema/39_assigner_action_list.sql

```diff
@@ -18,34 +18,42 @@ CREATE FUNCTION assigner_list (stage_id int, assigner_id int)
     AS $$
     SELECT
         review_assignment.application_id AS application_id,
-        CASE
-            WHEN COUNT(DISTINCT (review_assignment.id)) != 0 
-                AND assigned_questions_count(application_id, $1, level_number) >= assignable_questions_count(application_id)
-                AND submitted_assigned_questions_count(application_id, $1, level_number) < assigned_questions_count(application_id, $1, level_number)
-                THEN 'RE_ASSIGN'
-            WHEN COUNT(DISTINCT (review_assignment.id)) != 0 
-                AND assigned_questions_count(application_id, $1, level_number) >= assignable_questions_count(application_id)
-                AND submitted_assigned_questions_count(application_id, $1, level_number) >= assigned_questions_count(application_id, $1, level_number)
-                THEN 'ASSIGN_LOCKED'
-            WHEN COUNT(DISTINCT (review_assignment.id)) != 0 
-                AND assigned_questions_count(application_id, $1, level_number) < assignable_questions_count(application_id)
-                THEN 'ASSIGN'
-            ELSE NULL
+        CASE WHEN COUNT(DISTINCT (review_assignment.id)) != 0
+            AND assigned_questions_count (application_id, $1, level_number) >= assignable_questions_count (application_id)
+            AND submitted_assigned_questions_count (application_id, $1, level_number) < assigned_questions_count (application_id, $1, level_number) THEN
+            'RE_ASSIGN'
+        WHEN COUNT(DISTINCT (review_assignment.id)) != 0
+            AND assigned_questions_count (application_id, $1, level_number) >= assignable_questions_count (application_id)
+            AND submitted_assigned_questions_count (application_id, $1, level_number) >= assigned_questions_count (application_id, $1, level_number) THEN
+            'ASSIGN_LOCKED'
+        WHEN COUNT(DISTINCT (review_assignment.id)) != 0
+            AND assigned_questions_count (application_id, $1, level_number) < assignable_questions_count (application_id) THEN
+            'ASSIGN'
+        ELSE
+            NULL
```

Comment:
This is all just formatting changes, right?

Reply:
Yea LOL